app/models/project.rb

```diff
@@ -139,4 +141,12 @@ def retired_subjects_count
   def finished?
     @finished ||= workflows.all?(&:finished?)
   end
+
+  def state
+    if self[:state]
+      super
+    else
+      launch_approved ? "live" : "development"
```

Comment:
hmm - we've already got a live flag on projects...seems we're conflating this flag here. Just because a project is not launch approved (goes on the home page) doesn't mean it's not "live". I think we should probably just use the live flag itself here instead of `launch_approved` (you can't be launch approved without being live).